src/bin/merchant/manage.rs

```diff
@@ -48,25 +52,29 @@ impl Command for Show {
         // TODO: don't hard-code XTZ here, instead store currency in database
         let amount = |b: u64| Amount::from_minor_units_of_currency(b.try_into().unwrap(), XTZ);
 
-        let mut table = Table::new();
-        table.load_preset(comfy_table::presets::UTF8_FULL);
-        table.set_header(vec!["Key", "Value"]);
-        table.add_row(vec![Cell::new("Channel ID"), Cell::new(details.channel_id)]);
-        table.add_row(vec![Cell::new("Status"), Cell::new(details.status)]);
-        table.add_row(vec![
-            Cell::new("Contract ID"),
-            Cell::new(details.contract_id),
-        ]);
-        table.add_row(vec![
-            Cell::new("Merchant Deposit"),
-            Cell::new(amount(details.merchant_deposit.into_inner())),
-        ]);
-        table.add_row(vec![
-            Cell::new("Customer Deposit"),
-            Cell::new(amount(details.customer_deposit.into_inner())),
-        ]);
+        if config.json {
+            println!("{}", serde_json::to_string(&details).unwrap());
```

Comment:
Non-blocking: Could you use early-return here? (& also in src/bin/customer/manage.rs)